### PR TITLE
Update README with hot deploy

### PR DESCRIPTION
**Pull request by jhadvig**
Since php picks up changes in source by default, there is nothing else just update README && official docs.

@mfojtik @bparees thoughts ?

### CODE REVIEW

**Comment by mfojtik** [comment]
Please say where the sources are located inside the container, that will be the first question you will get on IRC when somebody will try to use hot-deploy ;-)

**Comment by jhadvig** [comment]
@mfojtik updated :mountain_cableway:

**Comment by bparees** [comment]
[test]

File: README.md [hunk 1]
```diff
@@ -211,3 +211,16 @@ In case the **DocumentRoot** of the application is nested within the source dire
 users can provide their own **.htaccess** file.  This allows the overriding of Apache's behavior and
 specifies how application requests should be handled. The **.htaccess** file needs to be located at the root
 of the application source.
+
+Hot deploy
+---------------------
+
+Hot deploy works in the built image out of the box.
```
Comment (bparees) [inline README.md]: "Hot deploy works in this image out of the box."

**Pull request merged**